lib/flop/schema.ex

```diff
@@ -249,7 +249,7 @@ defprotocol Flop.Schema do
 
       Owner
       |> join(:left, [o], p in assoc(o, :pets), as: :pets)
-      |> preload([pets: p], [pets: p])
+      |> preload([pets: p])
```

Comment:
I think it was supposed to be this:

```suggestion
      |> preload([p], [pets: p])
```